packages/plugin/src/rules/graphql-js-validation.ts

```diff
@@ -62,35 +63,57 @@ const validationToRule = (
     }
   }
 
+  const requiresSchema = docs.requiresSchema ?? true;
+
   return {
     [name]: {
       meta: {
-        ...meta,
         docs: {
           category: 'Validation',
-          requiresSchema: !skipSchema,
+          requiresSchema,
           requiresSiblings: false,
           url: `https://github.com/dotansimha/graphql-eslint/blob/master/docs/rules/${name}.md`,
-          ...meta.docs,
+          ...docs,
           description:
-            meta.docs.description +
+            docs.description +
             `\n\n> This rule is a wrapper around a \`graphql-js\` validation function. [You can find it's source code here](https://github.com/graphql/graphql-js/blob/main/src/validation/rules/${ruleName}Rule.ts).`,
         },
       },
-      create: context => {
+      create(context) {
+        let documentNode: DocumentNode;
+
+        const isGraphQLImportFile = rawSDL => {
```

Comment:
make it's better export and import this function ? https://github.com/ardatan/graphql-tools/blob/master/packages/loaders/graphql-file/src/index.ts#L35